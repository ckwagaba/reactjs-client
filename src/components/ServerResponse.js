import React, { Component } from 'react';

class ServerResponse extends Component {
  render () {
    return (
      <div className="server_response">
        {this.props.serverResponse}
      </div>
    );
  }
}

export default ServerResponse;
