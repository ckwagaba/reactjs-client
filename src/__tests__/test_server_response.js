import React from 'react';
import { shallow } from 'enzyme';
import ServerResponse from '../components/server_response';

describe('ServerResponse component', () => {

  const component = shallow(
    <ServerResponse />
  );

  it('should exist', () => {
    expect(component).toMatchSnapshot();
  });

});
