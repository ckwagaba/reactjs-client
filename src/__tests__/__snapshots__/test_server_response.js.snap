// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ServerResponse component should exist 1`] = `<ServerResponse />`;
